.circleci/common.sh

```diff
@@ -0,0 +1,2 @@
+MINICONDA_VER=4.5.4
```

Comment:
please don't include extra files

Reply:
I had to merge .circleci/common.sh from bioconda-recipes:master in order for the pull request to run the CI tests - I didn't touch a character in that file.

Reply:
I think you are confusing things, there is no common.sh on master
https://github.com/bioconda/bioconda-recipes/tree/master/.circleci

you probably downloaded it (from bioconda-common repo) to execute the tests.

Reply:
Ok. I wandered down several rabbit holes trying to set up CircleCI and the bootstrap testing frameworks locally and must have picked it up somewhere along the line... I'll get rid of it and recommit.

Reply:
no worries, there is a curl command on .circleci/config.yml that downloads that file. thats probably how you got it.